src/ol-ext/interaction/measure.js

```diff
@@ -25,6 +26,45 @@ goog.require('ol.style.Style');
 ngeo.interaction.MeasureBaseOptions;
 
 
+/**
+ * @enum {string}
+ */
+ngeo.MeasureEventType = {
+  /**
+   * Triggered upon feature draw end
+   * @event ol.MeasureEvent#end
+   */
+  END: 'end'
```

Comment:
I'd have named this measureend for consistency with ol3 (which has drawstart and drawend), but it's up to you.